doc/architecture/decisions/0020-test-coverage-strategy.md

```diff
@@ -0,0 +1,29 @@
+# 20. test-coverage-strategy
+
+Date: 2020-01-13
+
+## Status
+
+Accepted
+
+## Context
+
+Looking at our waning test coverage, we decided we needed a review of our test coverage
+strategy, especially a consideration of which parts of the project should be included
+in test coverage reports.
+
+## Decision
+
+* If low-level code is consistently coming up as uncovered, first check to
+  see if anything in the project is actually using that code. If not, remove
+  it or at least comment it out.
```

Comment:
I don't think I agree with leaving unused/untested code commented out. (Unless there is a very good reason in a specific case to do so.  We have a lot of extra code, a lot if it's boilerplate and would be easy to reimplement if we end up needing it. I'd say either delete it  -- or add tests and keep.

Reply:
Updated to eliminate the option to comment it out.